geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/cluster/ClusterExecutor.java

```diff
@@ -129,6 +129,18 @@ private RedisResponse getNodes(ExecutionHandlerContext ctx) throws InterruptedEx
     return RedisResponse.bulkString(response.toString());
   }
 
+  private Map<String, List<Integer>> getMemberBuckets(
+      List<SlotAdvisor.MemberBucketSlot> bucketSlots) {
+    Map<String, List<Integer>> result = new HashMap<>();
+
+    for (SlotAdvisor.MemberBucketSlot mbs : bucketSlots) {
+      result.computeIfAbsent(mbs.getMember().getUniqueId(), k -> new ArrayList<>())
+          .add(mbs.getBucketId());
+    }
```

Comment:
Previously, we were creating members if they didn't exist for a particular bucketID... do we no longer need to do that?

Reply:
Ah, nevermind, I see that it happens when we call `getBucketSlots`